user-registration-app/src/app/views/registration-form/registration-form.component.css

```diff
@@ -4,6 +4,12 @@
   margin-left: 35%;
   padding-bottom: 25px;
 }
+
+.registration-title{
+  text-align: center;
+  font-family: 'Courier New'
```

Comment:
I think this would look better with our CornerStone font, it's the font we generally use for headings like this.

Reply:
will do!